src/site/server/index.markdown

```diff
@@ -5,43 +5,170 @@ description: "Information on server-side programming for Dart."
 * [Tutorial: Write HTTP Clients & Servers](/docs/tutorials/httpserver/)
-* [Articles on a variety of Dart subjects](/articles/),
-   many written by Dart engineers
-* [Programmers Guide: Command-Line Apps](/docs/serverguide.html)&mdash;a
-  collection of links to examples, tools, and other documentation
-  of interest to server-side programmers
+* [Walkthrough of a working sample server][walkthrough] complete with
+  HTTP server, WebSockets, logging, and routing
+
+### Get support
+
+Join our
+[cloud@dartlang.org](https://groups.google.com/a/dartlang.org/forum/#!forum/cloud)
+mailing list and chat with other engineers building
+server-side Dart apps.
+
+Post your "how-to" questions to
+[Stack Overflow](http://stackoverflow.com/tags/dart).
+
+And of course, we welcome [bug reports](http://dartbug.com/new).
+
+### Libraries
 
-### Tell me more about the services that Dart supports.
+There are lots of options for building server-side and command-line
+apps with Dart. Here are just some of our favorites.
 
-Check out the following for information about services that Dart supports. 
+[dart:io][dart_io_libs] provides core functionality, such as files, directories,
+sockets, HTTP, and lots more. [Shelf][shelf] is a middleware system for
+web servers, with a robust plugin system. [Redstone][redstone] is a
+server-side microframework for Dart. [MySQL driver][mysql] helps you 
+connect to MySQL from Dart. [PostgreSQL driver][postgres] helps you
+connect to PostgreSQL from Dart. [Args][args] is a configurable parser of
+command-line args. [Http][http] helps you easily connect to HTTP resources.
 
-* [Dart and Google Cloud Platform](google-cloud-platform/)
-* [Google APIs Client Libraries with Dart](/googleapis/)
+Check [pub.dartlang.org][pub] for more options and libraries, including
+ORM, Redis, Mongo, authentication, Google APIs, and more.
 
-### Where can I host my Dart server?
+### Learn more
+
+Watch this video, recorded at the [Dart Developer Summit](/events/2015/summit/),
+about moving from Node.js to Dart.
+
+<iframe width="640" height="360" src="https://www.youtube.com/embed/NHsmiY0rFS8?list=PLOU2XLYxmsIIQorIS8gagUiMau9S84vZV&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+Then, dive deep with our [articles](/articles/), examples,
+and other documentation.
+
+* [Articles on a variety of Dart subjects](/articles/),
+   many written by Dart engineers
+* [Dart by Example][examples] has tons of small code snippets for servers
+  and consoles
+
+### Hosting Dart servers
 
 You have many options. Here are a few:
 
 * Google Cloud Platform; for details on Dart support, see
   [Dart and Google Cloud Platform](google-cloud-platform)
+* [Heroku](https://github.com/igrigorik/heroku-buildpack-dart)  
 * [Dartvoid](http://www.dartvoid.com/)
-* [Amazon Elastic Compute Cloud (EC2)](http://aws.amazon.com/ec2/)
 
+In addition, Dart runs on most cloud providers, such as
+Amazon's EC2, Digital Ocean, and more.
+
```

Comment:
Add if you are using Docker take a look at the base images povided (https://registry.hub.docker.com/u/google/dart/, https://registry.hub.docker.com/u/google/dart-runtime-base/ and https://registry.hub.docker.com/u/google/dart-runtime/.